 Description: Disable failing tests
 Disable tests that fail on Ubuntu pending investigation
Author: Vladimir Petko <vladimir.petko@canonical.com>
Forwarded: not-needed
Last-Update: 2023-03-24
--- a/test/hotspot/jtreg/ProblemList.txt
+++ b/test/hotspot/jtreg/ProblemList.txt
@@ -38,6 +38,26 @@
 #
 #############################################################################
 
+# tests that need to be investigated
+
+gc/shenandoah/mxbeans/TestChurnNotifications.java#aggressive 0000000 generic-all
+gc/shenandoah/mxbeans/TestChurnNotifications.java#iu 0000000 generic-all
+
+#result: Failed. Execution failed: `main' threw exception: java.io.IOException: Mount point not found
+# https://bugs.openjdk.org/browse/JDK-8166162
+
+runtime/LoadClass/LongBCP.java 8166162 generic-all
+
+# Intermittent failures because LingeredApp.class is not found
+
+runtime/cds/appcds/jcmd/JCmdTestDynamicDump.java 000000 generic-all
+runtime/cds/appcds/jcmd/JCmdTestFileSafety.java  000000 generic-all
+runtime/cds/appcds/jcmd/JCmdTestStaticDump.java  000000 generic-all
+
+# Unexpected exit from the test
+
+runtime/NMT/SafepointPollingPages.java 000000 generic-all
+
 # :hotspot_compiler
 
 compiler/aot/verification/vmflags/TrackedFlagTest.java 8215224 generic-all
--- a/test/langtools/ProblemList.txt
+++ b/test/langtools/ProblemList.txt
@@ -23,6 +23,32 @@
 #
 ###########################################################################
 
+# Wrong test environment in autopkgtest
+# java.lang.IllegalStateException: Launching JShell execution engine threw: ERROR: transport error 202: getaddrinfo: failed to parse address
+
+jdk/jshell/JdiListeningLocalhostExecutionControlTest.java 0000000 generic-all
+
+# tests failing due to disable-doclint-by-default.diff
+
+jdk/javadoc/doclet/testSupplementary/TestSupplementary.java 0000000 generic-all
+jdk/javadoc/doclet/testPackageHtml/TestPackageHtml.java 0000000 generic-all
+jdk/javadoc/doclet/testSummaryTag/TestSummaryTag.java 0000000 generic-all
+jdk/javadoc/doclet/testStylesheet/TestStylesheet.java 0000000 generic-all
+jdk/javadoc/doclet/testSeeTag/TestSeeTag.java 0000000 generic-all
+jdk/javadoc/doclet/testRelativeLinks/TestRelativeLinks.java 0000000 generic-all
+jdk/javadoc/doclet/testBadHtml/TestBadHtml.java 0000000 generic-all
+jdk/javadoc/doclet/testParamTaglet/TestParamTaglet.java 0000000 generic-all
+jdk/javadoc/doclet/testNonInlineHtmlTagRemoval/TestNonInlineHtmlTagRemoval.java 0000000 generic-all
+jdk/javadoc/doclet/dupThrowsTags/TestDupThrowsTags.java 0000000 generic-all
+jdk/javadoc/doclet/testLinkOption/TestLinkOption.java 0000000 generic-all
+jdk/javadoc/doclet/T6735320/T6735320.java 0000000 generic-all
+jdk/javadoc/doclet/testValueTag/TestValueTag.java 0000000 generic-all
+jdk/javadoc/tool/modules/Modules.java 0000000 generic-all
+jdk/javadoc/doclet/testWarnings/TestWarnings.java 0000000 generic-all
+jdk/javadoc/doclet/testHtmlTableStyles/TestHtmlTableStyles.java 0000000 generic-all
+jdk/javadoc/tool/doclint/DocLintTest.java 0000000 generic-all
+jdk/javadoc/doclet/testBadPackageFileInJar/TestBadPackageFileInJar.java 0000000 generic-all
+
 ###########################################################################
 #
 # javadoc
--- a/test/jdk/ProblemList.txt
+++ b/test/jdk/ProblemList.txt
@@ -110,6 +110,49 @@
 #
 #############################################################################
 
+
+
+# to investigate
+# unable to read LD_LIBRARY_PATH env variable
+
+tools/jpackage/share/AppLauncherEnvTest.java 0000000 generic-all
+
+# java.lang.Exception: Proc abnormal end
+
+sun/security/krb5/auto/Cleaners.java 0000000 generic-all
+
+# result: Failed. Execution failed: `main' threw exception: java.io.IOException: Mount point not found
+# https://bugs.openjdk.org/browse/JDK-8166162
+
+java/io/File/createTempFile/TargetDirectory.java 8166162 generic-all
+java/nio/file/Files/CheckPermissions.java 8166162 generic-all
+java/nio/file/Files/TemporaryFiles.java 8166162 generic-all
+java/nio/file/Files/Misc.java 8166162 generic-all
+java/nio/file/Files/CopyAndMove.java 8166162 generic-all
+java/nio/file/Files/InterruptCopy.java 8166162 generic-all
+java/nio/file/Files/FileAttributes.java 8166162 generic-all
+java/nio/file/attribute/BasicFileAttributeView/SetTimesNanos.java 8166162 generic-all
+java/nio/file/attribute/AclFileAttributeView/Basic.java 8166162 generic-all
+java/nio/file/attribute/DosFileAttributeView/Basic.java 8166162 generic-all
+java/nio/file/attribute/PosixFileAttributeView/Basic.java 8166162 generic-all
+java/nio/file/attribute/UserDefinedFileAttributeView/Basic.java 8166162 generic-all
+java/nio/file/FileStore/Basic.java 8166162 generic-all
+java/nio/channels/FileChannel/directio/DirectIOTest.java 8166162 generic-all
+java/nio/channels/FileChannel/directio/ReadDirect.java 8166162 generic-all
+java/nio/channels/FileChannel/directio/PreadDirect.java 8166162 generic-all
+java/nio/channels/FileChannel/directio/PwriteDirect.java 8166162 generic-all
+java/nio/channels/FileChannel/directio/WriteDirect.java 8166162 generic-all
+
+# utf-8 locale is missing on build machine
+# https://bugs.openjdk.org/browse/JDK-8249079
+
+java/lang/invoke/lambda/LambdaFileEncodingSerialization.java 8249079 generic-all
+
+# reproducible-properties-timestamp.diff introduce a bug that breaks this test
+# remove after the patch is dropped
+
+java/util/logging/LogManager/Configuration/updateConfiguration/SimpleUpdateConfigWithInputStreamTest.java 000000 generic-all
+
 ############################################################################
 
 # jdk_awt
